bin/build

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+npm i
+npm i -g lerna
+lerna bootstrap
+npm link
```

Comment:
I find this suspicious, why are we npm-linking loopback-next to the global module directory?